Source/WebKit/NetworkProcess/PrivateClickMeasurement/PrivateClickMeasurementManager.cpp

```diff
@@ -336,6 +336,12 @@ void PrivateClickMeasurementManager::getSignedUnlinkableTokenForDestination(Sour
 
 void PrivateClickMeasurementManager::insertPrivateClickMeasurement(PrivateClickMeasurement&& measurement, PrivateClickMeasurementAttributionType type, CompletionHandler<void()>&& completionHandler)
 {
+    if (m_client->usesEphemeralDataStore()) {
+        ASSERT(measurement.isEphemeral() == WebCore::PCM::AttributionEphemeral::Yes);
+        m_pendingEphemeralClickMeasurement = WTFMove(measurement);
+        return completionHandler();
+    }
+
     store().insertPrivateClickMeasurement(WTFMove(measurement), type, WTFMove(completionHandler));
```

Comment:
We should make sure store().insertPrivateClickMeasurement() release asserts if it's ever called in an ephemeral session.